ui/app/components/app/transaction-list/transaction-list.container.js

```diff
@@ -5,24 +5,52 @@ import TransactionList from './transaction-list.component'
 import {
   nonceSortedCompletedTransactionsSelector,
   nonceSortedPendingTransactionsSelector,
+  getTxParams,
 } from '../../../selectors/transactions'
-import { getSelectedAddress, getAssetImages } from '../../../selectors/selectors'
+import { getSelectedAddress, getAssetImages, getFeatureFlags } from '../../../selectors/selectors'
 import { selectedTokenSelector } from '../../../selectors/tokens'
 import { updateNetworkNonce } from '../../../store/actions'
+import { fetchBasicGasAndTimeEstimates, fetchGasEstimates } from '../../../ducks/gas/gas.duck'
+import {
+  getBasicGasEstimateBlockTime,
+  getCustomGasPrice,
+  getEstimatedGasPrices,
+  getEstimatedGasTimes,
+} from '../../../selectors/custom-gas'
+import { getRenderableTimeEstimate } from '../../../helpers/utils/gas-time-estimates.util'
+import { hexWEIToDecGWEI } from '../../../helpers/utils/conversions.util'
+
+const mapStateToProps = (state, ownProps) => {
+  const { selectedAddressTxList } = state.metamask
+  const { modalState: { props: modalProps } = {} } = state.appState.modal || {}
+  const { transaction = {} } = ownProps
+  const { txData = {} } = modalProps || {}
+  const selectedTransaction = selectedAddressTxList.find(({ id }) => id === (transaction.id || txData.id))
+  const { gasPrice: currentGasPrice } = getTxParams(state, selectedTransaction)
+  const customModalGasPriceInHex = getCustomGasPrice(state) || currentGasPrice
+  const customGasPrice = calcCustomGasPrice(customModalGasPriceInHex)
+  const gasPrices = getEstimatedGasPrices(state)
+  const estimatedTimes = getEstimatedGasTimes(state)
+  const transactionTimeFeatureActive = getFeatureFlags(state).transactionTime
 
-const mapStateToProps = state => {
   return {
     completedTransactions: nonceSortedCompletedTransactionsSelector(state),
     pendingTransactions: nonceSortedPendingTransactionsSelector(state),
     selectedToken: selectedTokenSelector(state),
     selectedAddress: getSelectedAddress(state),
     assetImages: getAssetImages(state),
+    blockTime: getBasicGasEstimateBlockTime(state),
```

Comment:
This prop appears to be unused

Reply:
The `customGasPrice` prop just below this appears to be unused as well

Reply:
@danjm : It looks like you may have missed this comment? These two unused props are still here.

Reply:
haha, no I made the change just didn't save the file, I've pushed an update now